iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/functionclass/ServerSimulator.java

```diff
@@ -92,22 +113,38 @@ public void run() {
         final Scheduler scheduler = new Scheduler(device);
         device.initalizeDevice(serverWrapper);
 
+        // Comparing database with model
+
+        DatabaseUtils databaseUtils = new DatabaseUtils(serverWrapper);
+        databaseUtils.setRelayRepository(this.relayRepository);
+        databaseUtils.setScheduleRepository(this.scheduleRepository);
+        for (int i = 1; i < device.getRelays().length + 1; i++) {
+            Relay relay = device.getRelay(i);
+            databaseUtils.checkRelay(relay);
+
+            for (int j = 1; j < 51; j++) {
```

Comment:
Cleaner: "<= 50" instead of "<51"
More meaningful: "scheduleNumber" instead of j.

Reply:
Done